build/blocks/add-to-calendar/render.php

```diff
@@ -25,12 +25,12 @@
 					<div class="gp-add-to-calendar__list" style="display: none;">
 						<?php foreach ( $gatherpress_event->get_calendar_links() as $gatherpress_calendar ) : ?>
 							<div class="gp-add-to-calendar__list-item">
-								<?php if ( ! empty( $gatherpress_calendar['link'] ) ) : ?>
+							<?php if ( ! empty( $gatherpress_calendar['link'] ) ) : ?>
 								<a href="<?php echo esc_url( $gatherpress_calendar['link'] ); ?>" target="_blank" rel="noopener noreferrer">
 									<?php elseif ( ! empty( $gatherpress_calendar['download'] ) ) : ?>
 									<a href="<?php echo esc_attr( $gatherpress_calendar['download'] ); ?>" rel="noopener noreferrer">
-										<?php endif; ?>
-										<?php echo esc_html( $gatherpress_calendar['name'] ); ?>
+									<?php endif; ?>
+							<?php echo esc_html( $gatherpress_calendar['name'] ); ?>
```

Comment:
Indenting of conditionals on this file were a bit messed up before but this is still a bit messed up.